docs/configuration/http_conn_man/http_conn_man.rst

```diff
@@ -106,6 +107,16 @@ http_codec_options (Warning: DEPRECATED and will be removed in 1.4.0)
   <config_cluster_manager_cluster_http_codec_options>` option. See the comment there about
   disabling HTTP/2 header compression.
 
+.. _config_http_conn_man_http1_settings:
```

Comment:
IMO it would be good to get a brief description of forward proxy support in the arch overview. Maybe here? https://lyft.github.io/envoy/docs/intro/arch_overview/http_routing.html (with links to here).